### ISSUE TITLE
Patched version in microsoft windows

### ISSUE THREAD

**Issue by lcorsini** [gabrielelana/awesome-terminal-fonts#12]
I'm stuck with windows at work, I tried your patched branch, but without luck, Mintty refuses to see the patched fonts, there is no chance to use them or there is another way to patch them?

**Comment by gabrielelana** [comment]
@lcorsini I don't know what to say, I don't use windows since '95, but I'll leave this issue open in case someone comes with an answer :smile:

**Comment by lcorsini** [comment]
I would be glad to ditch windows but I can't atm.
The problem seems to be that Mintty desn't see the font as monospaced so it doesn't show in mintty options

**Comment by DanielGGordon** [comment]
There are ways to trick windows into thinking it is monospaced. I haven't quite figured that out yet.

**Comment by DanielGGordon** [comment]
The thing is I haven't even been able to get the regular powerline patched fonts to recognize all symbols. And I haven't seen anyone who has.

**Comment by oneezy** [comment]
I'm trying to do this too for the all new **Windows Linux Subsystem**. 
I'll update my progress here.

-----

**In the meantime, here are some useful resources:**
- https://www.herebedragons.io/bashing-windows
- https://www.reddit.com/r/Windows10/comments/4orqn6/setting_up_powerline_for_bash_on_windows/
- http://askubuntu.com/questions/283908/how-can-i-install-and-use-powerline-plugin?newreg=c6b15c8c64e24e3fac09b8ac411b2807

**Comment by lcorsini** [comment]
The Linux subsystem should be an unmodifed ubuntu Linux userspace, so if
you install an X-server and a graphical terminal like terminator you could
use fontconfig fallback strategy

Il giorno 3 gennaio 2017 @ 11:07:24, oneezy (notifications@github.com) ha
scritto:

**Comment by oneezy** [comment]
I installed the [ConEmu Terminal](https://conemu.github.io/) and it's looking 1000x better! 
Well, a LOT better then what it was. The icon fonts still aren't showing but some of the special characters are. Still messing around w/ it...

But ConEmu is amazing.

**Comment by Naramsim** [comment]
@oneezy Did you managed to install these icon fonts? With ConEmu?

**Comment by oneezy** [comment]
Not yet @naramsim

But I'll be sure to report back here when i do

**Comment by superbob** [comment]
As a side note, you can use patch fonts on mintty (installed along with MSYS2).

However, it will not give you the option to choose the font in the configuration GUI as windows does not consider it to be a "monospace" font.

But, you can still choose it by manually edit the .minttyrc configuration file.

Anyway the renderring is really bad, I've openned an issue (#37) regarding that.

**Comment by DanielGGordon** [comment]
@oneezy So that's ConEmu using the new Linux Subsystem for Windows? That's cool - although I've heard you cannot use the arrow keys in vim while using ConEmu.

**Comment by oneezy** [comment]
@DanielGGordon yes, that's what the screenshot is. I haven't tested the arrow keys in vim yet

**Comment by DanielGGordon** [comment]
@oneezy How is the speed? If you just press `Enter` and get the terminal back - how long does that take? When I had `zsh` on Babun (Windows 7), it would take 2-3 seconds sometimes. When I was doing this, I remember that Meslo was one of the only fonts that worked correctly. I'm hopefully upgrading my Windows 7 work laptop to Windows 10 today and should give this a whirl.

**Comment by oneezy** [comment]
@DanielGGordon , I found ZSH to be pretty slow, but further investigating led me to realizing it had a lot to do with the `order` i wrote the settings in.

Also, I went through **a LOT of trial and error** to get the Windows Subsystem for Linux working to mimic my previous development workflow. You'll need to get in the [Windows Insider Program](https://insider.windows.com/) _(it's free)_ and so you can install the most recent build that allows for interopability _(the current version of WSL doesn't have it)_. 

-----

There's a great article you should read on this topic called [Bashing Windows](https://www.herebedragons.io/bashing-windows) that gives the exact approach I took _(read my comments below also)_.

Good luck! Let me know how it turns out for you.
